gssapi/krb5Token.go

```diff
@@ -155,11 +154,6 @@ func NewAuthenticator(creds credentials.Credentials, keyType int, flags []int) (
 	if err != nil {
 		return auth, krberror.Errorf(err, krberror.KRBMsgError, "error generating new authenticator")
 	}
-	etype, err := crypto.GetEtype(keyType)
-	if err != nil {
-		return auth, krberror.Errorf(err, krberror.KRBMsgError, "error getting etype for authenticator")
-	}
-	auth.GenerateSeqNumberAndSubKey(keyType, etype.GetKeyByteSize())
```

Comment:
As you mention in your PR comment the NewAuthenticator method now has a pointless keyType argument. I think we should update the comment on this to be something like: ```// NewAuthenticator (DEPRECATED - this method will be updated in future versions to remove the obsolete keyType argument and may be made private to the gssapi package) creates a new kerberos authenticator for kerberos MechToken```